sdk/network/mgmt/src/main/java/com/azure/management/network/ExpressRouteCircuitPeering.java

```diff
@@ -121,16 +121,16 @@
              *     same VLAN ID
              * @return next stage of definition
              */
-            WithPeerAsn withVlanId(int vlanId);
+            WithPeerASN withVlanId(int vlanId);
         }
 
         /** The stage of Express Route Circuit Peering definition allowing to specify AS number for peering. */
-        interface WithPeerAsn {
+        interface WithPeerASN {
             /**
-             * @param peerAsn AS number for peering. Both 2-byte and 4-byte AS numbers can be used
+             * @param peerASN AS number for peering. Both 2-byte and 4-byte AS numbers can be used
              * @return next stage of definition
              */
-            WithCreate withPeerAsn(long peerAsn);
+            WithCreate withPeerASN(long peerASN);
```

Comment:
Does this pass checkstyle?

Reply:
Yes, the checkstyle doesn't add ASN in blacklist.